From 37d732f459a62a8874d640599a7d142ffcb5619c Mon Sep 17 00:00:00 2001
From: Finley Xiao <finley.xiao@rock-chips.com>
Date: Tue, 22 Sep 2020 22:45:09 +0800
Subject: [PATCH] dt-bindings: power: add binding for rk3568 power domains

Add binding documentation for the power domains
found on Rockchip RK3568 SoCs.

Signed-off-by: Finley Xiao <finley.xiao@rock-chips.com>
Change-Id: I7544a643096948e75f59fb8126201b0de142440d
---
 .../devicetree/bindings/soc/rockchip/power_domain.txt          | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Documentation/devicetree/bindings/soc/rockchip/power_domain.txt b/Documentation/devicetree/bindings/soc/rockchip/power_domain.txt
index dbaeb7e8ee88..957af14ebea8 100644
--- a/Documentation/devicetree/bindings/soc/rockchip/power_domain.txt
+++ b/Documentation/devicetree/bindings/soc/rockchip/power_domain.txt
@@ -16,6 +16,7 @@ Required properties for power domain controller:
 	"rockchip,rk3366-power-controller" - for RK3366 SoCs.
 	"rockchip,rk3368-power-controller" - for RK3368 SoCs.
 	"rockchip,rk3399-power-controller" - for RK3399 SoCs.
+	"rockchip,rk3568-power-controller" - for RK3568 SoCs.
 - #power-domain-cells: Number of cells in a power-domain specifier.
 	Should be 1 for multiple PM domains.
 - #address-cells: Should be 1.
@@ -34,6 +35,7 @@ Required properties for power domain sub nodes:
 	"include/dt-bindings/power/rk3366-power.h" - for RK3366 type power domain.
 	"include/dt-bindings/power/rk3368-power.h" - for RK3368 type power domain.
 	"include/dt-bindings/power/rk3399-power.h" - for RK3399 type power domain.
+	"include/dt-bindings/power/rk3568-power.h" - for RK3568 type power domain.
 - clocks (optional): phandles to clocks which need to be enabled while power domain
 	switches state.
 - pm_qos (optional): phandles to qos blocks which need to be saved and restored
@@ -116,6 +118,7 @@ The index should use macros in:
 	"include/dt-bindings/power/rk3366-power.h" - for rk3366 type power domain.
 	"include/dt-bindings/power/rk3368-power.h" - for rk3368 type power domain.
 	"include/dt-bindings/power/rk3399-power.h" - for rk3399 type power domain.
+	"include/dt-bindings/power/rk3568-power.h" - for rk3568 type power domain.
 
 Example of the node using power domain:
 
-- 
2.35.3

